projects/baby_driver/scripts/test_can_send.py

```diff
@@ -0,0 +1,37 @@
+"""This Module Tests methods in can_send.py"""
+# pylint: disable=unused-import
+
+import unittest
+from unittest.mock import patch
+
+import can
+import can_util
+import cantools
+from can_send import can_send_raw, load_dbc, can_send
+
+
+class TestCanSendRaw(unittest.TestCase):
+    """Tests functions in Babydriver's can_send module"""
+
+    @patch('can_util.send_message')
+    def can_send_raw(self, mock_send_message):
+        """Tests that can_send_raw raises an Exception if CAN msg fails to send"""
+        mock_send_message.return_value = can.CanError
```

Comment:
This would just return the class `can.CanError` from `send_message` (which you can do: classes are first-class objects in Python and even have their own classes, "metaclasses"!) - what you can do is set `side_effect` to `can.CanError` and that'll throw CanError.